src/utils/util.ts

```diff
@@ -0,0 +1,13 @@
+import fetch from 'node-fetch';
+
+module.exports.openInteractiveDialog = async (dialog) => {
```

Comment:
Why do you need to open an interactive dialog from the api? It should be openable from the call response.

Reply:
removing. This was leftover from early testing.